src/components/GlobalFilter/GlobalFilter.tsx

```diff
@@ -129,17 +182,16 @@ const GlobalFilter = ({ hasAccess }: { hasAccess: boolean }) => {
 
 const GlobalFilterWrapper = () => {
   const [hasAccess, setHasAccess] = useState(false);
-  const globalFilterRemoved = useSelector(({ globalFilter: { globalFilterRemoved } }: ReduxState) => globalFilterRemoved);
+  const globalFilterRemoved = useAtomValue(globalFilterHiddenAtom);
   const chromeAuth = useContext(ChromeAuthContext);
   const { pathname } = useLocation();
   const { getUserPermissions } = useContext(InternalChromeContext);
 
   // FIXME: Clean up the global filter display flag
   const isLanding = pathname === '/';
   const isAllowed = isGlobalFilterAllowed();
-  const globalFilterHidden = useSelector(({ globalFilter: { globalFilterHidden } }: ReduxState) => globalFilterHidden);
-  const activeModule = useAtomValue(activeModuleAtom);
-  const isDisabled = globalFilterHidden || !activeModule;
+
+  const isDisabled = useAtomValue(isDisabledAtom);
```

Comment:
The name of this atom `isDisabledAtom` is very generic. It can be interpreted that any element can be disabled with this. Can you rename it to match its purpose better please?